lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -3206,7 +3209,7 @@ DefConstructor('\@@generic@caption[]{}', "<ltx:text class='ltx_caption'>#2</ltx:
 
 sub add_subfigure_row_breaks {
   my ($document, $node, $whatsit, @contents) = @_;
-  my $textwidth     = (LookupRegister('\textwidth') || Dimension('345pt'))->valueOf();
+  my $textwidth     = $whatsit->getProperty('floatwidth') || Dimension('345pt')->valueOf();
```

Comment:
Good news, the issue with the `1307.6769` listings breaks was a small typo on this line. We need to get the numerical value behind the `floatwidth` property, either when setting it, or simply changing this line to:
```perl
   my $textwidth     = ($whatsit->getProperty('floatwidth') || Dimension('345pt'))->valueOf();
```

Reply:
Hah! Good catch! How'd it even appear to work with that??

Reply:
perl being perl...

```
$ perl -e 'print "LaTeXML::Common::Dimension(whatever)" > 0 ? "bigger!\n" : "not bigger!\n"'
not bigger!
```

Reply:
Or: LaTeXML! It works for *all* the wrong reasons!